obs_youtube_uploader/ui/api.py

```
@@ -1132,6 +1246,291 @@ def auth_labels(self) -> dict:
+        """
+        if self._preview_host is None:
+            return
+        section = self._state.settings.get("preview", {})
+        # Pushed before start(): the first registration pass runs inside
+        # start(), and a table applied only after it would leave every
+        # binding unregistered until the next explicit save.
+        self._preview_host.set_hotkeys(section.get("hotkeys") or {})
+        if section.get("enabled"):
+            self._preview_host.start()
+
+    def set_preview_enabled(self, enabled: bool) -> None:
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)